_posts/newsletters/2019-10-24-newsletter.md

```diff
@@ -0,0 +1,155 @@
+discussions about topics suggested by participants. Sandra Gesing
+suggested a learning lab about US-RSE and the lively discussion was
+mainly around how US-RSE can support career paths in the US and help
+with creating positions at institutions. One take-away is that
+templates for job descriptions would be helpful for managers offering
+RSE-like positions.
+
+## Talk at 4th UK RSE Conference
+
+As part of the “[RSE Worldwide: Sharing across
+borders](https://rseconuk2019.sched.com/event/QP6S/rse-worldwide-sharing-across-borders)”
+plenary session at [RSEConUK2019](https://rse.ac.uk/conf2019/), Daniel
+S. Katz presented a lightning talk entitled “[US-RSE: Growing the
+Community](https://static.sched.com/hosted_files/rseconuk2019/6a/US%20RSE%20RSEconUK2019%20Lightning%20Talk.pdf)”
+and was part of the session’s panel discussion. The audience was
+particularly interested in discussion about the scope of RSEs in the
+US, as well as how we were growing and who we included.  
+
+## Upcoming meetings 
+
+### SC19 - Nov. 17-22, 2019 
+
+If you’re planning to attend SC19, there will be two panels and a BOF
+involving RSE leaders from the US and UK, and many other sessions of
+interest to RSEs.  We’re also working on arranging an informal US-RSE
+meeting (or more than one?)  during the conference - details TBA.  For
+more information see the [US-RSE
+blog](https://us-rse.org/2019-10-04-SC19/).  We hope you can join us!
+
+### Community Call 
+
+The next US-RSE community call is set for Tuesday, October 29, 2019 at
+2pm ET. Looking forward to talking with you all!
+
+Follow the [US-RSE
+calendar](https://calendar.google.com/calendar/embed?src=kgh79lg13k1d8a2o1s6megfuhc%40group.calendar.google.com&ctz=America%2FNew_York)
+of events to stay up to date.
+
+## Code of Conduct
+
+We now have an official [US-RSE code of
+conduct](https://us-rse.org/code-of-conduct/). Thanks to everyone who
+contributed. If you haven’t seen the final version, please take a few
+minutes to review it.
+
+## Recent Job Postings
+
+These opportunities were recently posted to the [RSE Careers page](https://us-rse.org/jobs/):
+[Scientific Research Software Developer](https://chm.tbe.taleo.net/chm03/ats/careers/v2/viewRequisition?org=CALTECH&cws=37&rid=4938): Caltech/JPL/Dartmouth
+[Research Software Engineer - FTC](https://careers.peopleclick.com/careerscp/client_arm/external/en_GB/gateway.do?functionName=viewFromLink&localeCode=en-us&jobPostId=43583&source=LinkedIn&sourceType=NETWORKING_SITE): Warwick/UK
+[Research Software Engineer/Senior Research Software Engineer](https://brown.wd5.myworkdayjobs.com/en-US/staff-careers-brown/job/180-George-Street/Research-Software-Engineer-Senior-Research-Software-Engineer_REQ162388): Brown University
+
+## RSE Stories
+
+The [Research Software Engineer
+Stories](https://us-rse.org/rse-stories/) is a new interview-style
+podcast designed to spotlight the diverse and interesting people
```

Comment:
Episode 4 is mostly done and 5 is underway! For 5 I’m testing an actual storytelling format, the goal to also be able to tel the stories of those that might be too introverted to talk on a podcast. Stay tuned!